### ISSUE TITLE
User Story:  Make UX (navigation and look-and-feel) consistent within financial_planner site 

### ISSUE THREAD

**Issue by DeeMcCart** [DeeMcCart/CI_PP4_Financial_Planner#36]
EPIC:  UX

 SO04 As **site owner** I want to **provide straightforward, intuitive, consistent website navigation** so that users can easily retrieve financial information/guidance which is relevant to them

### Assumptions:
Site has been developed with deliverable functionality in place
Look and feel is currently inconsistent or inadequate on site pages

### Acceptance Criteria

* Use graphical navigation where possible,including when accessing text-based informataion
* Consistent appearance of forms
* Consistent navigation

### Tasks
- [x] Task0.1 MyPlanner page - would like to click back on parent article to review its content before taking an action
- [x] Task0.2 MyPlanner Page - reviewing the site 14/11/23, 'delete' option for bookmarks not really needed - can un-like or un-bookmark directly from article detail
- [x] Task1 Change image display (rounded corners & number of articles on index page, opacity reduced to 75%)
- [x] Task2 Message confirm on user actions
- [x] Task3 Content - test adding video clips to content
- [ ] Task4 Confirm/cancel button needed when adding user action (ie can go backward if needed)
- [x] Task5 On reviewing the site 15/11/23, visibility of article create date is not particularly useful, as users want to see the date last updated..... add new field 'updated_on' to article model and make it visible
- [x] Task6 on reviewing the site 15/11/23, realised that user actions shoudl really have an optional 'url' field, as many actions require clickthrough to a next step, and currently the user needs to return to the article to get the required URL. 
- [ ] Task7 When a user takes like/unlike bookmark/unbookmark from the index page, the article detail page is displayed - should refresh and redisplay the index page instead
- [x] Task8 Change background colours on MyPlanner page as too bright on smaller devices
- [ ] Task9 Think abut page layout on MyPlanner page, when this is displayed on tableet (portrait) there's too much content 
- [x] Task10 Update icons (hands-clap or thumbs-up rather than 'heart' for like); tooltips above icons; 'Respond' rather than 'Comment'; 'Save' & 'Reading List' rather than bookmark
- [x] Task11 Add message confirmation for like/unike and for added/removed to reading list
- [x] Task12 After like/unlike confirmed, focus goes to top of page, should return to same location as previously
- [ ] Task13 Alter comment display appearance below article
- [x] Task14 Various issues with user profile functionaity, logged as a separate issue  as additional work needed
- [x] Task15 Add user avatar to navbar
- [ ] Task16 404 page

**Comment by DeeMcCart** [comment]
15/11/23 IN PROGRESS:  Ticking off the tasks on the list and adding new items as they come about - maintaining an issue log in Excel to track the individual observations in parallel with this.  Any larger items needing corrective work will be raised as github project bugs.
